src/main/java/com/fundation/logic/model/search/ImageSearch.java

```diff
@@ -67,6 +67,8 @@ public List searchInPath(String path) {
                 } else {
                     String fileName = FileInfo.getFileDenomination(fileExtractor, "name");
                     String fileExtension = FileInfo.getFileDenomination(fileExtractor, "extension");
+                    String mimeType = FileInfo.getMimeType(fileExtractor);
```

Comment:
mimeType should be retrieved using exiftool

Reply:
done